src/main/java/gyro/aws/kms/KmsKeyMultiRegionConfiguration.java

```diff
@@ -0,0 +1,65 @@
+package gyro.aws.kms;
+
+import gyro.aws.Copyable;
+import gyro.core.resource.Diffable;
+import software.amazon.awssdk.services.kms.model.MultiRegionConfiguration;
+import software.amazon.awssdk.services.kms.model.MultiRegionKey;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class KmsKeyMultiRegionConfiguration extends Diffable implements Copyable<MultiRegionConfiguration> {
+
+    private String multiRegionKeyType;
+
+    private KmsKeyMultiRegionKey primaryKey;
+
+    private List<KmsKeyMultiRegionKey> replicaKeys;
+
+    @Override
+    public void copyFrom(MultiRegionConfiguration model) {
+        KmsKeyMultiRegionKey multiRegionKey = newSubresource(KmsKeyMultiRegionKey.class);
+        multiRegionKey.copyFrom(model.primaryKey());
+
+        setPrimaryKey(multiRegionKey);
```

Comment:
We want to reset the value if it's not present upstream

```suggestion
        setPrimaryKey(null);
        if (model.primaryKey() != null) {
            KmsKeyMultiRegionKey multiRegionKey = newSubresource(KmsKeyMultiRegionKey.class);
            multiRegionKey.copyFrom(model.primaryKey());
            setPrimaryKey(multiRegionKey);
        }
```